src/main/java/com/werfen/report/service/pdf/PdfFormReportService.java

```diff
@@ -14,20 +14,32 @@
 import net.sf.jasperreports.engine.JasperPrint;
 import net.sf.jasperreports.engine.design.JasperDesign;
 
+import java.io.ByteArrayOutputStream;
 import java.io.File;
 
 public class PdfFormReportService {
 
     public File export(FormReportConfiguration formReportConfiguration, FormReportData formReportData, PageFormat pageFormat) throws JRException {
```

Comment:
In the same line as other comments, I would rename this method to be exportToFile